pkg/deployment/config.go

```diff
@@ -116,6 +121,52 @@ func defaultConfig() *Config {
 	return cfg
 }
 
+func (d Config) PodRuntimeClassName(lbs map[string]string) *string {
+	cfg := map[string]RuntimeClassNameLabelSelector{}
+	if err := yaml.Unmarshal([]byte(d.RuntimeClassName), &cfg); err != nil {
+		return nil
+	}
```

Comment:
We should do this parsing when constructing the config map - then if the label selector isn't valid the user will get feedback when applying the changes.

Reply:
@dprotaso, good thinking! I've updated the parsing to happen in NewConfigFromMap.